pytest_django/plugin.py

```diff
@@ -644,8 +644,9 @@ def _fail_for_invalid_template_variable():
     class InvalidVarException:
         """Custom handler for invalid strings in templates."""
 
-        def __init__(self) -> None:
+        def __init__(self, *, origin_value: str = "") -> None:
```

Comment:
I don't think the default is needed

```suggestion
        def __init__(self, *, origin_value: str) -> None:
```